docs/platforms/react-native/sourcemaps/uploading/codepush.mdx

```diff
@@ -26,25 +26,26 @@ export default codePush(Sentry.wrap(App));
 To ensure Sentry can symbolicate events from your CodePush releases, you need to generate and upload the necessary assets. When creating a CodePush release, include the `--sourcemap-output-dir` flag to generate source maps. This allows you to upload these files to Sentry in the next step.
 
 ```bash {tabTitle:JSC}
-appcenter codepush release-react \
-  --app "${APP_NAME}" \
-  --deployment-name "${DEPLOYMENT_NAME}" \
-  --output-dir ./build \
-  --sourcemap-output-dir ./build
+code-push-standalone release-react \
```

Comment:
`l`: AppCenter Codepush will be sunset March 31st, so there might still be ppl using the old command, right? does it make sense to keep both in tabs, with the new command being the default?

Reply:
Yes, till end of March users can use the app center commands. I'll return them back for the transition period and we can remove them afterwards.